tests/test_ijit/test_ijit.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+import numba
```

Comment:
We cannot expect `numba` to be installed on all systems. I recommend using `mock` [like the other test does](https://github.com/einsteinpy/einsteinpy/blob/c3498327a3f0aca5595eeb0ed452c8350d0a544e/tests/test_ijit/test_ijit_without_numba.py#L10). These tests have to assume, `numba` isn't available.

That aside, you are on the right track, but you would be missing out on Line 17 this way:
![image](https://user-images.githubusercontent.com/31299799/103775561-3913b580-5054-11eb-9bb1-dc8b67e08b1a.png)

Also, instead of adding another file, you can rename the other test file to `test_ijit.py` and keep both these tests in the same file. Just rename the other test to `test_ijit_no_numba` and call this `test_ijit_return` or something better.

Reply:
And, I'd remove [L23](https://github.com/einsteinpy/einsteinpy/blob/c3498327a3f0aca5595eeb0ed452c8350d0a544e/tests/test_ijit/test_ijit_without_numba.py#L23) from the other test. It's redundant.